js/page-specific/site-search-results.js

```diff
@@ -30,7 +31,7 @@ bcpl.pageSpecific.swiftypeSearchResults = (($, querystringer, Handlebars, consta
 	const getSearchResults = (searchTerm, pageNumber) => {
```

Comment:
not blocking but it might be nice to get the url stuff out of here, and into it's own function.